robot/camera.py

```diff
@@ -48,7 +48,7 @@ def _cam_listener_worker(self):
 
         Works until `self._running` is set.
         """
-        while self._running.is_set():
+        while not self._stop.is_set():
```

Comment:
This isn't especially clear, particularly with the comment above referring to the old name. Maybe introduce a property `alive` (or some similar name) which is the used like:
``` python
while self.ailve:
    ...
```
The definition of the property can then be separately documented and hide the excessive negation inside it.